### PR TITLE
Pay 4938 new attribute

### PR DESCRIPTION
**Pull request by Jennings-HMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/FeeVersionDto.java [hunk 1]
```diff
@@ -3,14 +3,12 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import lombok.*;
```
Comment (vigneshpushkaranHMCTS) [inline api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/FeeVersionDto.java:6]: Replace  * with neccesary imports

File: api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/ReasonDto.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.fees2.register.api.contract;
+
+import lombok.*;
```
Comment (vigneshpushkaranHMCTS) [inline api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/ReasonDto.java:3]: Replace * with necessary imports

Comment (Jennings-HMCTS) [reply api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/ReasonDto.java]: chanes done

File: api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/FeeVersionDto.java (hunk 1 above)
Comment (Jennings-HMCTS) [reply api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/FeeVersionDto.java]: changes done

**Review by vigneshpushkaranHMCTS — APPROVED** [review]

**Pull request merged**